browser-test/src/application_navigation.test.ts

```diff
@@ -0,0 +1,88 @@
+import {AdminPrograms, AdminQuestions, ApplicantQuestions, loginAsAdmin, loginAsGuest, logout, selectApplicantLanguage, startSession, resetSession} from './support'
+
+describe('applicant navigation flow', () => {
+  let pageObject;
+
+  beforeAll(async () => {
+    const {page} = await startSession()
+    pageObject = page;
+  });
+
+  afterEach(async () => {
+    await resetSession(pageObject);
+  });
+
+  describe('navigation with four blocks', () => {
```

Comment:
Since this is now the test file that tests navigating through an application, we should also add tests to ensure that "Next" button works. This would be a good check to add anyway since we're introducing the "Previous" logic, we should ensure that "next" still works. 

You can add a separate test for this, or just check it within your existing tests (i.e. add checks after each .clickNext()). If you choose the latter, remember to modify the test names.

Reply:
Yeah, I'll add this as a follow-up if that's ok?

Reply:
Sure! Perhaps add a TODO to this file to keep track.

Reply:
Good call, added!